lab-matt/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-05-tcp-chat-server",
+  "version": "0.0.1",
+  "description": "Local Chat App",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/muddybeard210/lab-05-tcp-chat-server.git"
+  },
+  "author": "Matthew Sheridan-Smith",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/muddybeard210/lab-05-tcp-chat-server/issues"
+  },
+  "homepage": "https://github.com/muddybeard210/lab-05-tcp-chat-server#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
You shouldn't have 'chai', 'mocha', or 'gulp-mocha' because you don't have any tests.